ansible/roles/debops.php/defaults/main.yml

```diff
@@ -83,7 +83,7 @@ php__server_api_packages: [ 'cli', 'fpm' ]
 # .. envvar:: php__base_packages [[[
 #
 # Install set of standard PHP packages.
-php__base_packages: [ '{{ "php" + php__version }}', 'curl', 'gd', 'mcrypt' ]
+php__base_packages: [ '{{ "php" + php__version }}', 'curl', 'gd', '{{ "mcrypt" if php__version | version_compare("7.2","<") else None }}' ]
```

Comment:
For better readability, I would convert this to a vertical list. Also, using `None` like this might create issues if/when APT will try to install a `None` package. I would change this to `[]`, the package list is passed through a `flattened` Ansible lookup which will remove an empty list. In effect, something like this:
```yaml
php__base_packages:
  - '{{ "php" + php__version }}'
  - 'curl'
  - 'gd'
  - '{{ "mcrypt"
        if (php__version | version_compare("7.2","<"))
        else [] }}'
```